### PR TITLE
syntax errors

### PR DESCRIPTION
**Pull request by reggi**

### CODE REVIEW

File: README.markdown [hunk 1]
```diff
@@ -661,7 +661,7 @@ Join an array into a human readable sentence.
 toSentence(["jQuery", "Mootools", "Prototype"]);
 // => "jQuery, Mootools and Prototype";
 
-toSentence(["jQuery", "Mootools", "Prototype"], ", ", " unt ');
+toSentence(["jQuery", "Mootools", "Prototype"], "", "", " unt ");
```
Comment (stoeffel) [inline README.markdown]: This should be:
```js
toSentence(["jQuery", "Mootools", "Prototype"], ", ", " unt ");

", " // is the delimiter
" unt " // is the lastDelimiter
```

File: README.markdown [hunk 2]
```diff
@@ -676,7 +676,7 @@ toSentenceSerial(["jQuery", "Mootools"]);
 toSentenceSerial(["jQuery", "Mootools", "Prototype"]);
 // => "jQuery, Mootools, and Prototype"
 
-toSentenceSerial(["jQuery", "Mootools", "Prototype"], ", ", " unt ');
+toSentenceSerial(["jQuery", "Mootools", "Prototype"], "", "", " unt ");
```
Comment (stoeffel) [inline README.markdown]: same here:
```js
toSentenceSerial(["jQuery", "Mootools", "Prototype"], ", ", " unt ");
```

**Comment by reggi** [comment]
Hey @stoeffel I applied the changes to the same commit and did a `--force` update on the `patch-1` branch.

**Comment by stoeffel** [comment]
Nice, thanks!

**Pull request merged**